modules/ROOT/pages/advance/logging.adoc

```diff
@@ -125,30 +189,106 @@ The default log level is `info`.
 |Displays extremely verbose output that might be useful when debugging
 |===
 
-
-[#log-keys]
+[#lbl-log-keys]
 === Log Keys
+TIP: Select log keys relevant to the area you are debugging, providing them as a comma-delimited list, such as: `"log_keys": ["HTTP", "CRUD", "Import"]` in the config or see <<eg-setloggingwithapi>> for how to provide them using the Admin REST API.
+
+Log keys provide fine-grained control over the information types that Sync Gateway outputs to the console log.
+By default, only *`HTTP`* related information is enabled, but a range of other keys are available to meet specific diagnostic needs -- see: <<tbl-logkeylist>>.
+
+You can define the required {xref-sgw-bmk-config-properties-log-console-key} within your configuration file and-or use the Admin REST API (see: <<eg-setloggingwithapi>>).
+
+.List of Available Log Keys
+[#tbl-logkeylist,cols="^2,8"]
+|===
+h|Log Key
+h|Description
```

Comment:
Quite a few of these log keys no longer exist (The Shadow log key for example):

Here's the full list of log keys in use in the current version:

https://github.com/couchbase/sync_gateway/blob/4b2567e1968973eab0330d07d06035cecc15848e/base/log_keys.go#L54-L75 

Let me know if you need any descriptions for more of them:
- `SGCluster` is related to the sharded import and HA sg-replicate